plugins/amongus/amongus.cpp

```diff
@@ -5,122 +5,132 @@
 
 #include "Game.h"
 
-#define MUMBLE_ALLOW_DEPRECATED_LEGACY_PLUGIN_API
-#include "../mumble_legacy_plugin.h"
-
-#include "../mumble_positional_audio_utils.h"
+#include "../MumbleAPI_v_1_0_x.h"
+#include "../MumblePlugin_v_1_0_x.h"
```

Comment:
I think the `plugins` directory should be added to the include path and these includes changes to
```suggestion
#include "MumblePlugin_v_1_0_x.h"
```

(Note also that `MumblePlugin_v_1_0_x.h` already includes `Mumble_API_v_1_0_x.h`

Reply:
Nah I think doing it in a separate commit is enough :)

Reply:
This is actually already the case: https://github.com/mumble-voip/mumble/blob/fa80b18a481b8c58e7e5c3a39c813cce1da1d70c/plugins/CMakeLists.txt#L44

In fact, the old headers are in the same directory and (legacy) plugins include them without going up one level.